Sources/Extensions/Foundation/DateExtensions.swift

```diff
@@ -952,3 +952,15 @@ public extension Date {
 	}
 	
 }
+
+extension Date: ExpressibleByIntegerLiteral {
+    /// SwifterSwift: Create date object from Int literal
+    ///
+    ///     let date = Date(inegerLiteral: 2017_12_25) // "2017-12-25 00:00:00 +0000"
+    /// - Parameter value: Int value, e.g. 20171225, or 2017_12_25 etc.
+    public init(integerLiteral value: Int) {
```

Comment:
I'd like this to be a failable initializer `init?(integerLiteral value: Int)`

Reply:
If we conform to ExpressibleByIntegerLiteral protocol, this init is predefined and it's not failable init. I've made it failable, removing protocol adoption.